doc/AUTHENTICATION.md

```diff
@@ -1,15 +1,26 @@
+# Authenticating against the AppFlowy API
+
+The AppFlowy API is in its infancy and provides experimental access to those who know how to deal with these things on a more basic level, until higher-level abstractions are available (API token generation in the app/web/console settings, autogenerated OpenAPI schema and public Swagger UI for an AppFlowy instance, etc.).
+
+Please proceed with caution.
+
 ## User Account
+
 For the purpose of automation, it is recommended to create a separate user account, then add the user as a member in the workspace.
 This is so that, in the event that the user account is compromised, the workspace owner can still revoke the user's access.
 
 ## Authentication
+
 AppFlowy Cloud employs token-based authentication. You can retrieve a refresh token via the following steps:
 
-1. Log in via the admin frontend. (https://beta.appflowy.cloud)
+1. Log in via the admin frontend. ([beta.appflowy.cloud/](https://beta.appflowy.cloud))
```

Comment:
I have forgotten to update this line for self hosters, where the admin frontend has been moved to /console. Will fix this in another PR.

Reply:
I have been using the access token provided by the `/web` app.